python/common/org/python/types/Float.java

```diff
@@ -355,7 +355,9 @@ public boolean __setattr_null(java.lang.String name, org.python.Object value) {
             } else {
                 throw new org.python.exceptions.ZeroDivisionError("float division by zero");
             }
-        }
+        }// else if (other instanceof org.python.types.Complex){
+           // return other.__truediv__(this);
+        //}
```

Comment:
Please don't add commented out code, these don't add anything useful and only confuse someone reading the code.

Reply:
Oh sorry for this. I have removed comments. I won't repeat this again and thanks.